src/server/migrations/13_blurhash.ts

```diff
@@ -19,9 +19,9 @@ export default class extends AbstractMigration {
     uploadedFilesDir: string
   ) {
     const filePath = path.join(uploadedFilesDir, object.filename);
-    console.log(`Calculating blurhash for ${filePath}`);
-    object.blurhash = await calculateBlurhash(filePath);
-    console.log(`Blurhash for ${filePath} is ${object.blurhash}`);
+    console.log(`Calculating blur hash for ${filePath}`);
```

Comment:
I would have thought that it is two words here, as we are referring to the concept "blur hash" which I would spell with two words, rather than the class itself?

Reply:
Hmm, it is spelled `BlurHash` on their GitHub repo and website (https://github.com/woltapp/blurhash), which is why I thought spelling it `BlurHash` would be ever so slightly more correct. But that's super nitpicky, feel free to to whatever you like best.

Reply:
Oh, and I just realized that this is part of a super old migration, which makes it even less important.

Reply:
but I mean if its how they write it, we might as well do it too :)